google-cloud-spanner/acceptance/spanner/client/transaction_test.rb

```diff
@@ -78,7 +80,7 @@
   end
 
   it "can rollback a transaction" do
```

Comment:
Yes, we can do something like [this unit test](https://github.com/blowmage/gcloud-ruby/blob/84a610bbe653a8a805c3584938d90d10ac903d67/google-cloud-spanner/test/google/cloud/spanner/client/transaction_test.rb#L134-L153).